core/domain/stats_domain.py

```diff
@@ -194,10 +314,15 @@ def get_sum_of_first_hit_counts(self):
             sum_first_hits += state_stats.first_hit_count
         return sum_first_hits
 
-    def validate(self):
+    def validate(self) -> None:
         """Validates the ExplorationStats domain object."""
 
-        exploration_stats_properties = [
+        exploration_stats_properties: List[Literal['num_starts_v1',
+            'num_starts_v2',
+            'num_actual_starts_v1',
+            'num_actual_starts_v2',
+            'num_completions_v1',
+            'num_completions_v2']] = [
```

Comment:
Can you try?

```suggestion
        exploration_stats_properties: Final = [
```

if that doesn't work do

```suggestion
        exploration_stats_properties: List[Literal['num_starts_v1',
            'num_starts_v2',
            'num_actual_starts_v1',
            'num_actual_starts_v2',
            'num_completions_v1',
            'num_completions_v2'
        ]] = [
```

Reply:
Done